tools/migrator/src/fixtures/user.ts

```diff
@@ -5,21 +5,28 @@ export const getUserFixtures: () => Insertable<Database["owUser"]>[] = () => [
   {
     auth0Sub: "auth0|4dd4698d-c376-4e5e-b5fb-4db9bf6cd417", // brage.baugerod@online.ntnu.no
     createdAt: new Date("2023-04-30 19:22:17.627253+00"),
+    email: "bragebaugerod@gmail.com",
+    givenName: "brage",
+    familyName: "baugertod",
+    name: "brage baugerod",
     studyYear: 3,
   },
   {
     auth0Sub: "auth0|7f6e0389-b49f-4bbc-b401-505f4b53fb3c", // njal.sorland@online.ntnu.no
     createdAt: new Date("2023-08-11 21:22:17.627253+00"),
+    email: "njalsorland@gmail.com",
+    givenName: "njal",
+    familyName: "sorland",
+    name: "njal sorland",
     studyYear: 5,
   },
   {
-    createdAt: new Date("2023-04-30 21:22:17.627253+00"),
-    auth0Sub: "auth0|dddddddd-c376-4e5e-b5fb-4db9bf6cd417",
-    studyYear: 0,
-  },
-  {
-    createdAt: new Date("2023-04-30 21:22:17.627253+00"),
-    auth0Sub: "auth0|7ba93bf4-2e11-40a9-bcd7-9078ee9090bb",
+    createdAt: new Date("2023-04-30 21:22:17.627253+00"), // henrikskog01@gmail.com
+    auth0Sub: "auth0|61ea20bf-6964-4484-b7b8-e81952606973",
+    email: "henrikskog01@gmail.com",
```

Comment:
IMO these should not be real names/emails, and we should look into how to get a good devX here.

Also these sub's are for dev?

Reply:
Yeah this setup is a bit awkward. My sub is for dev at least, I have not checked the others.

Reply:
I created a new issue for this on Linear. I don't think it's something that should be fixed as a part of this PR.